### PR TITLE
Tests for URLs.

### PR DESCRIPTION
**Pull request by damonkelley**

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @ldko @somexpert

**Comment by somexpert** [comment]
:+1:

File: tests/test_urls.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+def test_humanAgent_with_identifier():
+    url = resolve('/agent/agentId/')
+    assert url.func == views.humanAgent
+
+
+def test_agentXML():
+    url = resolve('/agent/agentId.xml')
+    assert url.func == views.agentXML
+
```
Comment (ldko) [inline tests/test_urls.py]: Should we have a test for '/agent/agentId.premis.xml'?

**Comment by ldko** [comment]
:+1:

**Pull request merged**